crates/sound/Cargo.toml

```diff
@@ -25,10 +25,10 @@ virtio-bindings = "0.2"
 virtio-queue = "0.7"
 vm-memory = "0.10"
 vmm-sys-util = "0.11"
-pipewire = { version = "0.6.0", optional = true }
-libspa = { version = "0.6.0", optional = true }
-pipewire-sys = { version = "0.6.0", optional = true }
-libspa-sys = { version = "0.6.0", optional = true }
+pipewire = { git = "https://gitlab.freedesktop.org/pipewire/pipewire-rs.git", rev = "a144cbd249ce4de34140c3e5e5c70490180cdc05", optional = true }
+libspa = { git = "https://gitlab.freedesktop.org/pipewire/pipewire-rs.git", rev = "a144cbd249ce4de34140c3e5e5c70490180cdc05", optional = true }
```

Comment:
Unrelated, but you don't need to add libspa/pipewire-sys/libspa-sys dependencies explicitly as they are re-exported by pipewire, see https://gitlab.freedesktop.org/pipewire/pipewire-rs/-/blob/main/pipewire/src/lib.rs#L119-120. So you could do `use pipewire::spa::SomeSpaType;`

Reply:
ok I'll try that thanks :)